src/main.rs

```diff
@@ -23,13 +25,28 @@ mod web;
 
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
-struct Args {
+pub struct Args {
     /// Override the port to host the web-ui and API on
     #[arg(short, long)]
     pub port: Option<u16>,
     /// Override the config file to use
     #[arg(short, long)]
     pub config: Option<String>,
+    /// Override the default tf2 directory
+    #[arg(short, long)]
+    pub tf2_dir: Option<String>,
+    /// Override the configured/default rcon password
+    #[arg(short, long)]
+    pub rcon_pword: Option<String>,
+    /// Override the configured Steam API key,
+    #[arg(short, long)]
+    pub api_key: Option<String>,
+    /// Whether to rewrite the user localconfig.vdf to append the new launch options,
+    #[arg(short = 'l')]
+    pub rewrite_launch_options: Option<bool>,
+    /// Whether to panic on detecting missing launch options
+    #[arg(short)]
+    pub no_panic_on_missing_launch_options: Option<bool>,
```

Comment:
Shorter name suggestion: `ignore_launch_options`

Reply:
`no_panic_on_missing_launch_options` -> `ignore_missing_launch_options`, arg short -> `i`, arg long -> `ignore_missing_lo` ?

Reply:
Hmm, I'm not a fan of the abbreviated "lo". But I also think it doesn't abbreviate to a short arg well, so many not having a short arg and just making the full name short enough, like "ignore_lo" or "ignore_launch_opts" would work.

Reply:
hmm i just feel that without `missing` we kind of lose the semantic value of CLA, and it becomes less obvious what its doing. Its already fairly vague because there is no indication to _which_ launch options we are ignoring

Reply:
could we be reasonable in equating launch options with CLAs, and just say `ignore_missing_cla`  or something?

Reply:
But we're talking about the game launch options, not CLAs for anything?

Reply:
arent launch options just CLA's for tf2 tho? :joy:

Reply:
> hmm i just feel that without `missing` we kind of lose the semantic value of CLA, and it becomes less obvious what its doing. Its already fairly vague because there is no indication to _which_ launch options we are ignoring

Well that's what the short description in the help menu is for. I personally think the "missing" is not necessary in the whole cla name, and that "launch options" is clearly enough referring to the game launch options.

Reply:
> arent launch options just CLA's for tf2 tho? joy clown_face (im reaching)

Well yeah technically, but who really be out here launch their TF2 from the command line?

Reply:
> Well yeah technically, but who really be out here launching their TF2 from the command line?
I guess that would be bot hosters?

> Well that's what the short description in the help menu is for. I personally think the "missing" is not necessary in the whole cla name, and that "launch options" is clearly enough referring to the game launch options.

true, and in thinking about it more, `ignore_launch_opts` is still technically doing that, not only are we ignoring the fact that some may be missing, we kind of just ignore them entirely